spec/unit/typesafe/config/path_spec.rb

```diff
@@ -0,0 +1,259 @@
+require 'spec_helper'
+require 'hocon'
+require 'test_utils'
+
+
+describe Hocon::Impl::Path do
```

Comment:
Does this test file correspond exactly to what's in the `PathTest` class in typesafe/config?  If so ... do we have any way of indicating that?  Is it just naming conventions?